### DISCUSSION TITLE
Custom scroll speed?

### DISCUSSION THREAD

**Discussion by BSteffaniak** [emilk/egui#5271] [Q&A] [answered]
Hi, I'm new to using egui. One thing I've noticed is that the scroll areas seem to scroll very noticeably slower than other native apps. Is there a way to customize the scroll speed for scroll areas?

**Comment by rustbasic** [comment]
See `Settings` of `egui_demo_app`.
See below :

```
                ui.ctx().all_styles_mut(|style| {
                    style.scroll_animation.points_per_second = 1000.0;
                    style.scroll_animation.duration.min = 0.1;
                    style.scroll_animation.duration.min = 0.3;
                });
```

**Reply by BSteffaniak** [reply]
That seems to only control the speed when you use something like `scroll_to_rect`:

```
/// The animation that should be used when scrolling a [`crate::ScrollArea`] using e.g. [Ui::scroll_to_rect].
pub scroll_animation: ScrollAnimation,
```

Is there a way to make that affect normal manual scrolling behavior?

**Comment by rustbasic** [comment, marked as answer]
See below :
```
                ui.ctx().options_mut(|options| {
                    options.line_scroll_speed = 40.0; // default
                });

                ui.ctx().options_mut(|options| {
                    options.line_scroll_speed = 100.0;
                });
```

**Reply by stravant** [reply]
Any consideration for changing the default?

As the original post states, the default is surprisingly slow: significantly slower than the system default on either Mac or Windows.
